README.md

```diff
@@ -11,33 +11,78 @@ The LUKSO CLI is a command line tool to install, manage and set up validators of
 - 📑 Accessing various node logs
 - 💰 Making validator deposits
 
-## Client Clarification
+## Supported EVM Clients
 
-> WIP: More client setups will be added
+> WIP: More clients will be added
 
-The LUKSO CLI is able to install multiple node clients.
-They cover the full node functionality of an EVM PoS Blockchain.
+The LUKSO CLI is able to install multiple clients for running the node.
 
-- Supported Execution Node Clients: [Geth](https://geth.ethereum.org/)
-- Supported Beacon Node Clients: [Prysm](https://github.com/prysmaticlabs/prysm)
+- Execution Node Clients: [Geth](https://geth.ethereum.org/)
+- Beacon Node Clients: [Prysm](https://github.com/prysmaticlabs/prysm)
 - Validator Client for Staking
```

Comment:
If we give links to each of those clients maybe we should also specify that the only validator we support right now is also provided by Prysmaticlabs?

Reply:
Indeed! will add that link